docs/rest_api/text_messages.rst

```diff
@@ -0,0 +1,372 @@
+Text messages
+#############
+
+Use this endpoint to obtain details on Mautic's Text Messages, or SMSes.
```

Comment:
```suggestion
Use this endpoint to obtain details on Mautic's Text Messages, or SMS's.
```

Reply:
I think it should be apostrophe'd but I am not 100% sure - open to discussion!

Reply:
There's lots of discussion about this [here](https://english.stackexchange.com/questions/51378/what-is-the-plural-form-of-sms), but the general consensus seems to be on `SMSs` or `texts`. WDYT?

Reply:
I'd go with SMSs as we refer to Text Messages immediately prior.

Reply:
Updated. Unfortunately the endpoint itself is `GET /smses/ID` so we can't update that part of the docs. It'd require a code update in `mautic/mautic` first.

Reply:
Yeah that will have to stay as it is for now! Thanks!